services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -74,7 +74,7 @@ async def _apply_observation_cycle(
         )
 
     try:
-        with timeout(
+        async with timeout(
```

Comment:
Now that I read it, was this blocking?

Reply:
The docs suggest this usage.
```
    >>> with timeout(0.001):
    ...     async with aiohttp.get('https://github.com') as r:
    ...         await r.text()
```

Reply:
you might see it in your own CI. unless this was due to some other update